Chapter1/Zad5.py

```diff
@@ -0,0 +1,38 @@
+'''
+Created on 28 sty 2018
+
+@author: agata.wiewiora
+'''
+
+
+def count_tables_amount (students_amount):
+    table_students = 2 
+    temp = students_amount % table_students
+    if temp > 0:
+        tables_amount = (students_amount // table_students) + 1
+    else:
+        tables_amount = students_amount // table_students
+    return tables_amount
+
+students = {}
+
+
+try:
+    # Reads how many students:
+    students_a = raw_input("Please, provide amount of students in class a: ")
```

Comment:
jak użyjesz input zamiast raw_input to nie będziesz musiała rzutować

Reply:
możesz przypisać raw_input bezpośrednio do students['students_A'], nie ma potrzeby tworzenia dodatkowej zmiennej